### DISCUSSION TITLE
Move from IRC to Discord

### DISCUSSION THREAD

**Discussion by Safihre** [sabnzbd/sabnzbd#2121] [Ideas]
It seems that Discord is the new hip thing to do. So should we indeed move there instead or IRC?
I've never used either, but nowadays I see Discord everywhere and I don't remember ever seeing IRC for any application. 😅
@thezoggy @jcfp opinions?

**Comment by thezoggy** [comment]
Sounds fine to me, but I'm on both already. 😜

**Comment by jeffgeorge** [comment]
Happy to try, but I have no idea what the level of effort is to effectively moderate a Discord server.

**Comment by Safihre** [comment]
Can we "remove" the IRC also, in case we want to switch?

**Reply by thezoggy** [reply]
i personally dont see a harm in keeping both. one can always just run a matrix server to bridge services and communicate to all from one spot if they didnt want to have to keep different clients.  but also no different to having github issues + sab forums + sab email. 

while on the topic, if we wanted to be more modern and move away from email/forums and have more disposable active workflow.. then irc+discord makes a bit more sense.

**Comment by jcfp** [comment]
Not a discord user myself, so not much to add other than practical considerations: the need or expectation of active moderation as mentioned by @jeffgeorge; is an account a hard requirement or is their some form of 'guest' access akin to mibbit; the centralized nature of the service.

**Reply by thezoggy** [reply]
Think of it like accessing the forums. People can chose if they want to create a throw away account or not..

While I understand why the purely guest/anonymous reasons why mibbit might be a nice idea, in practicality I've generally found them about the worst users to actually support. They ask a question and leave instantly. Have no clue how to share logs or do anything.
IRC for a lot of people that arent used to it (let alone if you dont have a proper client) while 'simple' does not work well for those wanting to get help. 

Using something like discord where images/links and things like bots/rules/guidelines and honestly a much larger user base would be welcomed. Especially in this day when a lot of people use MS teams/Slack for their job.. so using Discord is about the same.

**Comment by thezoggy** [comment]
went ahead and created sabnzbd discord and set it up. feel free to join
https://discord.gg/KQzDe7fvNU

**Comment by opentoe** [comment]
While moving to Discord sounds great it naturally turns into the same thing. A bunch if idlers, bots and no one really there.

**Reply by thezoggy** [reply]
even in the short amount of time sab discord has gone live, there has been far more activity on it than in irc. also with the bridge from irc<>discord thats been running, i've seen discord users helping out irc users when no one on irc is actually around.

**Comment by thezoggy** [comment]
@Safihre feel free to close this out
